data-es2016plus.js

```diff
@@ -3007,6 +3007,7 @@ exports.tests = [
         chrome66: true,
         duktape2_0: false,
         graalvm: true,
+        duktape2_3: false,
```

Comment:
It would be nice to keep all duktape entries together.

Reply:
Fixed!